cpp/ql/src/Critical/DescriptorMayNotBeClosed.ql

```diff
@@ -5,6 +5,7 @@
  * @id cpp/descriptor-may-not-be-closed
  * @problem.severity warning
  * @tags efficiency
+ *       security
```

Comment:
I added this for consistency with FileMayNotBeClosed.ql and FileNeverClosed.ql - it's arguable that the security tag should be removed in those queries instead of added here and in DescriptorNeverClosed.ql